tests/e2e/tests/pilot/testdata/pilot.yaml.tmpl

```diff
@@ -96,6 +96,10 @@ spec:
         - http://eureka:8080
 {{if .UseAdmissionWebhook}}
         - --admission-service={{.AdmissionServiceName}}
+{{end}}
```

Comment:
As you discovered, this is all gone - the changes need to be made in the install/k8s/helm directory, as they would be made for the release.